### PR TITLE
[Bug Fix] Fix GroupNorm Implementation

### PR DESCRIPTION
**Pull request by hgt312**
See https://github.com/apache/incubator-mxnet/issues/17139.

### CODE REVIEW

**Comment by sxjscience** [comment]
LGTM, FYI @Jerryzcn

File: python/mxnet/gluon/nn/basic_layers.py [hunk 1]
```diff
@@ -820,26 +820,29 @@ class GroupNorm(HybridBlock):
...
-                                     shape=(num_groups,), init=gamma_initializer,
+                                     shape=(in_channels,), init=gamma_initializer,
                                      allow_deferred_init=True)
         self.beta = self.params.get('beta', grad_req='write' if center else 'null',
-                                    shape=(num_groups,), init=beta_initializer,
+                                    shape=(in_channels,), init=beta_initializer,
                                     allow_deferred_init=True)
 
     def hybrid_forward(self, F, data, gamma, beta):
         norm_data = F.GroupNorm(data, gamma=gamma, beta=beta, num_groups=self._num_groups, eps=self._epsilon)
```
Comment (sxjscience) [inline python/mxnet/gluon/nn/basic_layers.py:838]: I just realized one quick issue. Should we consider to move GroupNorm to npx? Currently, the layer won’t be usable in the new numpy interface. @zhreshold

**Comment by sxjscience** [comment]
I've merged this PR because it's a bugfix. If we need GroupNorm in `npx`, we can register it under the `npx` namespace.

**Comment by yzhliu** [comment]
We should have a list of ops that need to be migrated.

**Pull request merged**